public/static/text/kn-mn.json

```diff
@@ -10588,6 +10588,10 @@
                         "type": "paragraph",
                         "text": "**සාතං අසාතන්ති යමාහු ලොකෙ**ති - ‘ සාත’න්ති සුඛා ච වෙදනා ඉට්ඨඤ්ච වත්ථු,{4} **අසාත**න්ති දුක්ඛා ච වෙදනා අනිට්ඨඤ්ච වත්ථු; **යමාහු ලොකෙ**ති යං ආහංසු යං කථෙන්ති යං භණන්ති යං දීපයන්ති යං වොහරන්තීති - ‘ සාතං අසාතන්ති යමාහු ලොකෙ.’"
                     },
+                    {
+                        "type": "paragraph",
+                        "text": "**තමූපනිස්සාය පහොති ඡන්දො**ති. සාතාසාතං නිස්සාය, සුඛදුක්ඛං නිස්සාය, සොමනස්සදොමනස්සං නිස්සාය, ඉට්ඨානිට්ඨං නිස්සාය, අනුනයපටිඝං නිස්සාය ඡන්දො පහොති පභවති ජායති සඤ්ජායති නිබ්බත්තති අභිනිබ්බත්තතීති - ‘ තමූපනිස්සාය පහොති ඡන්දො.’{*}"
+                    },
                     {
                         "type": "paragraph",
                         "text": "**රූපෙසු දිස්වා විභවං භවඤ්චා**’ති - ‘ රූපෙසූ’ති චත්තාරො ච මහාභූතා චතුන්නඤ්ච මහාභූතානං උපාදාය රූපං. කතමො රූපානං භවො? යො රූපානං භවො ජාති සඤ්ජාති නිබ්බත්ති අභිනිබ්බත්ති පාතුභාවො; අයං රූපානං භවො. කතමො රූපානං විභවො? යො රූපානං ඛයො වයො භෙදො පරිභෙදො අනිච්චතා අන්තරධානං; අයං රූපානං විභවො. **රූපෙසු දිස්වා විභවං භවඤ්චා**’ති රූපෙසු භවඤ්ච විභවඤ්ච දිස්වා පස්සිත්වා තුලයිත්වා තීරයිත්වා විභාවයිත්වා විභූතං කත්වාති - ‘රූපෙසු දිස්වා විභවං භවඤ්ච. ‘"
```

Comment:
Also note that `භවඤ්ච. ‘` has a left quotation mark, which needs a right quotation mark